<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTemplatesBase</class>
 <widget class="QWidget" name="CustomTemplatesBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layout9">
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="KLineEdit" name="mName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string comment="@info:tooltip">Enter a name for the custom template</string>
           </property>
           <property name="whatsThis">
            <string comment="@info:whatsthis">To create a custom template, first enter a name for that template and press the '+' button.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="mAdd">
           <property name="toolTip">
            <string comment="@info:tooltip">Add a new custom template</string>
           </property>
           <property name="whatsThis">
            <string comment="@info:whatsthis">Press this button if you want to create a new custom template with the specified name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="mRemove">
           <property name="toolTip">
            <string comment="@info:tooltip">Remove the selected custom template</string>
           </property>
           <property name="whatsThis">
            <string comment="@info:whatsthis">Press this button if you want to remove the currently selected custom template</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="mDuplicate">
           <property name="toolTip">
            <string comment="@info:tooltip">Duplicate the selected custom template</string>
           </property>
           <property name="whatsThis">
            <string comment="@info:whatsthis">Press this button if you want to duplicate the currently selected custom template</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="mList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">List of custom templates available</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">This is a list of all the custom templates available.</string>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string comment="@title:column Type of custom template">Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string comment="@title:column Name of the custom template.">Name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mHelp"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="mEditFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>12</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="mToLabel">
             <property name="text">
              <string comment="@label:textbox Recipient mail address">To:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mCCLabel">
             <property name="text">
              <string comment="@label">CC:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="MessageCore::EmailAddressRequester" name="mToEdit" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string comment="@info:tooltip">Additional recipients of the message</string>
             </property>
             <property name="whatsThis">
              <string comment="@info:whatsthis">When using this template, the default recipients are those you enter here.  This is a comma-separated list of mail addresses.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MessageCore::EmailAddressRequester" name="mCCEdit" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string comment="@info:tooltip">Additional recipients who get a copy of the message</string>
             </property>
             <property name="whatsThis">
              <string comment="@info:whatsthis">When using this template, the recipients you enter here will by default get a copy of this message.  This is a comma-separated list of mail addresses.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TemplateParser::TemplatesTextEdit" name="mEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">The custom template</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">This area contains the associated custom template.</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="1" column="2">
          <widget class="KComboBox" name="mType">
           <item>
            <property name="text">
             <string comment="@item:inlistbox Universal custom template type.">Universal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Custom template type for message replies">Reply</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Custom template type for reply-to-all messages">Reply to All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string comment="@item:inlistbox Custom template type for messsage forwards">Forward</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="textLabel1_2">
           <property name="text">
            <string comment="@label">Shortc&amp;ut:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mKeySequenceWidget</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="textLabel1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string comment="@label">&amp;Template type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>mType</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="KKeySequenceWidget" name="mKeySequenceWidget"/>
         </item>
         <item row="0" column="0">
          <widget class="TemplateParser::TemplatesInsertCommand" name="mInsertCommand" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TemplateParser::TemplatesTextEdit</class>
   <extends>QTextEdit</extends>
   <header>templatestextedit.h</header>
  </customwidget>
  <customwidget>
   <class>TemplateParser::TemplatesInsertCommand</class>
   <extends>QWidget</extends>
   <header>templatesinsertcommand.h</header>
  </customwidget>
  <customwidget>
   <class>MessageCore::EmailAddressRequester</class>
   <extends>QWidget</extends>
   <header>emailaddressrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
